src/react/DirtyFormStatePrompt.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import { constNull } from 'fp-ts/lib/function';
+import { declareQueries } from './declareQueries';
+// @ts-ignore: to avoid TS4023
```

Comment:
is there a nameSpace conflict?

Reply:
you can read a description of the TS4023 issue here https://github.com/Microsoft/TypeScript/issues/5711, as of today there are no "good" fixes if you have enabled also `noUnusedLocals` -> you have to ignore the unused type import otherwise TS would fail